### PR TITLE
nvs: add documentation about the crc in nvs

### PR DESCRIPTION
**Pull request by Laczen**
fixed nvs part of #68699

### CODE REVIEW

**Review by de-nordic — DISMISSED** [review]

File: doc/services/storage/nvs/nvs.rst [hunk 1]
```diff
@@ -19,7 +19,11 @@ combination of these.
 Each element is stored in flash as metadata (8 byte) and data. The metadata is
 written in a table starting from the end of a nvs sector, the data is
 written one after the other from the start of the sector. The metadata consists
-of: id, data offset in sector, data length, part (unused) and a crc.
+of: id, data offset in sector, data length, part (unused) and a crc. The crc is
+only calculated over the metadata and only ensures that a write has been
+completed. The data of the element is not protected by a crc. It is encouraged
+to include a crc as part of the data and to take appropriate corrective actions
+when the data crc does not match its expected value.
```
Comment (kartben) [inline doc/services/storage/nvs/nvs.rst:26]:
```suggestion
of: id, data offset in sector, data length, part (unused), and a CRC. The CRC is
only calculated over the metadata and only ensures that a write has been
completed. The actual data of the element is not protected by a CRC. It is
encouraged to include a CRC as part of the data and to take appropriate
corrective actions when the data CRC does not match its expected value.
```

**Review by kartben — DISMISSED** [review]
Thank you!
Just nits, really, non blocking.

**Review by kartben — APPROVED** [review]

**Comment by Laczen** [comment]
@kartben, it seems I couldn't just accept your commit.

**Comment by de-nordic** [comment]
> @kartben, it seems I couldn't just accept your commit.

You have to pull the PR to your local branch now and squash commits.

**Review by henrikbrixandersen — APPROVED** [review]

**Pull request merged**